HdProduction.HelpDesk.Api/Controllers/TicketsAdminController.cs

```diff
@@ -44,9 +45,10 @@ public async Task<TicketAdminResponseModel> Update(long id, TicketRequestModel r
     }
 
     [HttpPut("{id}/comment")]
-    public async Task<TicketAdminResponseModel> AddComment(long id, TicketRequestModel requestModel)
+    public async Task<TicketAdminResponseModel> AddComment(long id, CommentRequestModel requestModel)
```

Comment:
It should be POST method because we create a new entity instead of updating an old one.

Reply:
fixed